modules/markup/mdstripper/mdstripper.go

```diff
@@ -141,6 +141,7 @@ func (r *stripRenderer) AddOptions(...renderer.Option) {
 }
 
 // StripMarkdown parses markdown content by removing all markup and code blocks
+//
```

Comment:
Why does it add empty lines?

Reply:
I will fix these. This isn't a bug, per se, but it's because the next line has a deceiving `tab` instead of a `space`, which makes the go formatter think it's an indented code block, as seen on [pkg.go.dev](https://pkg.go.dev/code.gitea.io/gitea@v1.17.0/modules/markup/mdstripper).

Reply:
```suggestion
```